src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,33 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import './PostInfo.scss';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+export const PostInfo = ({ post }) => (
+  <>
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">
+          {post.title}
+        </h3>
+
+        <UserInfo user={post.user} key={post.user.id} />
```

Comment:
```suggestion
        <UserInfo user={post.user} />
```
we use key prop only in a loop

Reply:
user can be null, let's handle this case here